test/timing.c

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright 1995-2020 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ *
+ * I compiled this in the source tree this way:
+ *      gcc -g -Iinclude a.c libcrypto.a -ldl -lpthread
```

Comment:
a.c ?

Reply:
Year, that header comment is outdated and should be removed.

As written in the OP of https://github.com/openssl/openssl/issues/18814, one currently can use
```
gcc -g -I../include timing.c ../libcrypto.a -ldl -lpthread -o timing
```